build/common/worker/healthcheck.sh

```diff
@@ -1,28 +1,29 @@
 #!/bin/bash
+set -o errexit -o allexport -o nolog
```

Comment:
isn't `nolog` ignored by default?

can we instead make this:

```suggestion
set -ea
```

Reply:
Umm, yeah
> isn't nolog ignored by default?

sure, I just go with the verbose 
` set -ea`